tools/fast/series/ctx_mining_once.go

```diff
@@ -0,0 +1,28 @@
+package series
+
+import (
+	"context"
+)
+
+type ctxMiningOnceKey struct{}
+
+// Key used to store the MiningOnceFunc in the context
+var miningOnceKey = ctxMiningOnceKey{}
```

Comment:
It looks like this pattern of putting something in context is repeated a bit. Consider putting a single FAST interface object in the context with methods corresponding to the various contextual bits. Usage would then look like `ctxSeries(ctx).MineOnce()`

Reply:
Ya, I think when we add the price / limit we may move to this pattern.